Sharprompt/ConsoleRenderer.cs

```diff
@@ -7,10 +7,25 @@ internal class ConsoleRenderer
     {
         public ConsoleRenderer()
         {
+            Console.CancelKeyPress += (object o, ConsoleCancelEventArgs args) =>
```

Comment:
Since this event handler leaks after the `ConsoleRender` dispose, it is necessary to cancel the event at the time of `Close`.